### ISSUE TITLE
Final library missing name/version info

### ISSUE THREAD

**Issue by gdevenyi** [BIC-MNI/bicpl#3]
Not sure how this is solved, but while working on debian packaging of minc-toolkit-v2, get the following warning:
dpkg-shlibdeps: warning: can't extract name and version from library name 'libbicpl.so'

**Comment by vfonov** [comment]
Probably something like this : https://github.com/BIC-MNI/libminc/blob/master/CMakeLists.txt#L358 is required.

**Comment by gdevenyi** [comment]
Yup, based on a bit of googling, SOVERSION info is what is needed. See other issues on oobicpl mni_autoreg and libarguments
